SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorCommandLineArgsManager.h

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+#include "Templates/SharedPointer.h"
+
+class FReply;
+
+typedef TSharedRef<class ILauncher> ILauncherRef;
+typedef TSharedPtr<class ILauncherWorker> ILauncherWorkerPtr;
+typedef TSharedRef<class ILauncherProfile> ILauncherProfileRef;
+
+#if ENGINE_MAJOR_VERSION >= 4 && ENGINE_MINOR_VERSION >= 24
```

Comment:
out of curiosity: what will happen on UE 4.23? will it not work on that version?

Reply:
Yes, V4.23 hasn't that events, If we do it, we must change UE source code.

Reply:
we still support 4.23 as well, so we should support the removal of the UE4CommandLine.txt there as well

Reply:
I would say supporting this in 4.23 is not worth the extra engine changes. While we do technically support 4.23, this is mainly to allow existing customers to upgrade to 4.24, so I think if possible we should tell the new prospects to evaluate these features on 4.24.

Reply:
sure, I'm fine with that,. We might need to coordinate how to write that up in the documentation as usually we do add features to both engine versions, right?